src/m_alloc.cpp

```diff
@@ -52,14 +54,14 @@
 #endif
 #if defined(__APPLE__)
 #define _msize(p)				malloc_size(p)
-#elif defined(__sun)
+#elif __solaris__ || defined(__OpenBSD__)
```

Comment:
yes for having ported couple of soft to solaris it s not that simple (unfortunately).